docs/yaml.md

```diff
@@ -9,35 +9,37 @@ Each section adds file to the root file system
 
 ## `kernel`
 
-This section defines the kernel configuration. The `image` field specifies the Docker image,
+The `kernel` section defines the kernel configuration. The `image` field specifies the Docker image,
 which should contain a `bzImage` (for `amd64` architecture, others may vary) and a file
 called `kernel.tar` which is a tarball that is unpacked into the root, which should usually
 contain a kernel modules directory. See [`kernel/`](../kernel/) for source code. `cmdline`
 specifies the kernel command line options if required.
 
 ## `init`
 
-This section currently just lists images that is used for the `init` system and are unpacked directly
+The `init` section is a list of images that are used for the `init` system and are unpacked directly
 into the root filesystem. This should bring up `containerd`, start the system and daemon containers,
```

Comment:
you can do something totally different with linxukit, but its up to you to make it work...